.circleci/docker/common/install_base.sh

```diff
@@ -66,47 +66,79 @@ install_ubuntu() {
 install_centos() {
   # Need EPEL for many packages we depend on.
   # See http://fedoraproject.org/wiki/EPEL
-  yum --enablerepo=extras install -y epel-release
+  if [[ $OS_VERSION == 9 ]]; then
+      yum install -y epel-release
+  else
+      yum --enablerepo=extras install -y epel-release
+  fi
 
   ccache_deps="asciidoc docbook-dtds docbook-style-xsl libxslt"
   numpy_deps="gcc-gfortran"
   # Note: protobuf-c-{compiler,devel} on CentOS are too old to be used
   # for Caffe2. That said, we still install them to make sure the build
   # system opts to build/use protoc and libprotobuf from third-party.
-  yum install -y \
-    $ccache_deps \
-    $numpy_deps \
-    autoconf \
-    automake \
-    bzip2 \
-    cmake \
-    cmake3 \
-    curl \
-    gcc \
-    gcc-c++ \
-    gflags-devel \
-    git \
-    glibc-devel \
-    glibc-headers \
-    glog-devel \
-    hiredis-devel \
-    libstdc++-devel \
-    libsndfile-devel \
-    make \
-    opencv-devel \
-    sudo \
-    wget \
-    vim
-
+  if [[ $OS_VERSION == 9 ]]
+  then
+	  yum install -y --allowerasing \
```

Comment:
We can reduce this big if else block, by controlling 
`if [[ $OS_VERSION == 9 ]] ALLOW_ERASE = "--allowerasing"

Differing package `libsndfile-devel` can be in a specific block similarly.

For opencv, it seems CS9 package is not yet released.